servant-client/servant-client.cabal

```diff
@@ -35,24 +35,24 @@ library
     Servant.Common.BasicAuth
     Servant.Common.Req
   build-depends:
-      base >=4.7 && <5
-    , aeson
-    , attoparsec
-    , base64-bytestring
-    , bytestring
-    , exceptions
-    , http-api-data >= 0.1  && < 0.3
-    , http-client <0.5
-    , http-client-tls
-    , http-media
-    , http-types
-    , network-uri >= 2.6
-    , safe
-    , servant == 0.7.*
-    , string-conversions
-    , text
-    , transformers
-    , transformers-compat
+      base                  >= 4.7      && < 4.10
+    , aeson                 >= 0.7      && < 0.12
+    , attoparsec            >= 0.12     && < 0.14
+    , base64-bytestring     >= 1.0.0.1  && < 1.1
+    , bytestring            >= 0.10     && < 0.11
+    , exceptions            >= 0.8      && < 0.9
+    , http-api-data         >= 0.1      && < 0.3
+    , http-client           >= 0.4.18.1 && < 0.6
+    , http-client-tls       >= 0.2.2    && < 0.4
```

Comment:
If `http-client-tls` of some version doesn't compile with some version of its dependencies, it's problem of `http-client-tls`. I.e. downstream shouldn't worry about that. If there is a failure, I.e. Incorrect bounds, package maintainers or Hackage Trustees will correct that.